python-avd/scripts/build-schemas.py

```diff
@@ -117,7 +80,8 @@ def build_schema_tables(schema_store):
 
 def main():
     combine_schemas()
-    schema_store = compile_schemas()
+    print("Rebuilding pickled schemas")
```

Comment:
print? Why not a logger?

Reply:
This is just a cli script printing to pre-commit.